File: backend-model/src/main/java/outcobra/server/model/dto/SchoolClassDto.java
package outcobra.server.model.dto;

import outcobra.server.model.SchoolClass;
import outcobra.server.model.interfaces.MappableDto;
import outcobra.server.model.interfaces.Mapper;

public class SchoolClassDto implements MappableDto<SchoolClassDto, SchoolClass> {
    @Override
    public Mapper<SchoolClassDto, SchoolClass> getMapper() {
        throw new UnsupportedOperationException("not implemented");
    }

    @Override
    public SchoolClass toEntity() {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: backend/src/main/kotlin/outcobra/server/model/dto/emptyDto/SchoolClassDto.kt
package outcobra.server.model.dto.emptyDto

import outcobra.server.model.SchoolClass
import outcobra.server.model.mapper.MappableDto
import outcobra.server.model.mapper.Mapper

class SchoolClassDto : MappableDto<SchoolClassDto, SchoolClass> {
    override fun getMapper(): Mapper<SchoolClassDto, SchoolClass> {
        throw UnsupportedOperationException("not implemented")
    }

    override fun toEntity(): SchoolClass {
        throw UnsupportedOperationException("not implemented")
    }
//TODO Implement
}